tests/gradients/test_finite_difference.py

```diff
@@ -88,48 +88,6 @@ def test_correct_second_derivative_center_order4(self):
         assert np.allclose(shifts, [0, -1, 1, -2, 2])
 
 
-class TestShiftedTapes:

```

Comment:
maybe I'm missing it, but I can't see where this test has been moved to?

Reply:
As this is a helper method in `general_shift_rules.py`, it should be tested in `test_general_shift_rules.py` (`TestGenerateShiftedTapes`). There was, and currently still seems to be, some duplication. I'll look into it.

Reply:
Ah, I didn't find it because of the rename :)